test/cases/at-rules.out.css

```diff
@@ -0,0 +1,33 @@
+body {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
```

Comment:
I recommend enabling `cascade` here to make test readable and more standard

Reply:
I fixed that.